src/libp2p/Libp2p/Libp2pPeerFactoryBuilder.cs

```diff
@@ -27,7 +27,7 @@ protected override ProtocolStack BuildStack()
     {
         ProtocolStack tcpEncryptionStack = enforcePlaintext ?
             Over<PlainTextProtocol>() :
-            Over<NoiseProtocol>();
+            Over<TlsProtocol>();
```

Comment:
Should not be a replacement but an alternative to Noise, something like `.Over<NoiseProtocol>().Or<TlsProtocol>()`